contracts/evoting/evoting.go

```diff
@@ -854,6 +927,30 @@ func (e evotingCommand) manageAdminList(snap store.Snapshot, step execution.Step
 	return nil
 }
 
+// isRole check whether the txPerformingUser has the role in the provided form
+func (e evotingCommand) isRole(form types.Form, txPerformingUser string, role Role) (bool, error) {
+	sciperInt, err := types.SciperToInt(txPerformingUser)
+	if err != nil {
+		return false, xerrors.Errorf("Failed to convert SCIPER to int: %v", err)
+	}
+
+	if role == Voters {
+		for i := 0; i < len(form.Voters); i++ {
+			if form.Voters[i] == sciperInt {
+				return true, nil
+			}
+		}
```

Comment:
the list of voters can be quite long (e.g. 13 000 people at EPFL) - I wonder if it'd make sense to maintain the voters as a sorted list

that would make inserting new voters a bit more expensive, but looking up existing voters much less expensive

what do you think?